src/utilities/theme-utils.ts

```diff
@@ -21,6 +21,8 @@ export const colorShade = (color: string, amt: number) => {
 };
 
 export const setThemeColors = (primary: string, secondary: string) => {
+  const el = document.querySelector(".app") as HTMLElement;
+  el && el.style.setProperty("background-image",`linear-gradient(to bottom, #fff, rgba(255,255,255,0), rgba(255,255,255,0))`);
```

Comment:
Isn't this already set in the initial CSS?  It doesn't seem to do anything related to the theme colors so I don't think it belongs here (unless it's supposed to use on of the theme colors?).  Maybe this is supposed to come out?

Reply:
The color theme buttons only changes the colors on the top dom level. The gradient is set at the .app level. When I add the background image at the .app level when you click on the image button, it never changes back to the gradient when you click on the color theme buttons. This takes care of putting the gradient back in.

Reply:
Ah, so we lose the gradient if a background image is set and need to restore it.  What if an author wants to set a background image and theme colors?  Does this mean that they cannot do both?  It at least seems like this chunk should be in its own function since at some point we'll want to find a path to have both theme colors and a background color (and, at present, this would overwrite the background image which shouldn't be a side effect of calling a function that sets the theme colors).

Reply:
Yeah, perhaps a setGradient function should be added to the activity utils (or this might be an argument for an image-utils module that simply does:
```
const el = document.querySelector(".app") as HTMLElement;
  el && el.style.setProperty("background-image",`linear-gradient(to bottom, #fff, rgba(255,255,255,0), rgba(255,255,255,0))`);
```

Reply:
I think we want to keep it clean so that setting the theme colors only sets theme colors.  This would allow us, later on, to set a background image and a theme color (which presumably we will need to do) without causing conflict.

Reply:
Then when we use our test buttons, we would call both `setThemeColors` and `setBackgroundGradient` while still being able to call `setBackgroundImage` and `setThemeColors` in the future without the calls clashing.